libs/community/tests/unit_tests/test_vertex_ai_search.py

```diff
@@ -5,6 +5,7 @@
 from google.auth import credentials as ga_credentials
 from google.cloud.discoveryengine_v1beta import Document as DiscoveryEngineDocument
 from google.cloud.discoveryengine_v1beta.types import SearchRequest, SearchResponse
+from langchain_community.embeddings import FakeEmbeddings
```

Comment:
we try not to depend on langchain_community.
please, import `from langchain_core.embeddings import FakeEmbeddings` instead

Reply:
Make sense. Done!